tools/fw_zbbridge/readme.txt

```diff
@@ -1,3 +1,15 @@
-The ncp-uart-sw-6.8.0.1_115200.ota is the latest version for EZSP v8 compatible hosts
-The ncp-uart-sw_6.7.6_115200.ota is for EZSP v8 compatible hosts
-The ncp-uart-sw_6.5.5_115200.ota its for older like current ZHA
+# EmberZNet NCP UART EZSP firmware
+
+## EmberZNet NCP UART EZSP firmware signed for Sonoff ZBBridge
+
+The ncp-uart-sw-6.8.0.1_115200.ota is the latest development version, only testing with for EZSP v8 compatible hosts.
```

Comment:
Thanks. It would be good to add that v6.8 is still untested and that users shouldn't use it for now with Z2T.

Reply:
Updated readme.txt mentioning that 6.8.x.x is experimental and untested.